reference/docs-conceptual/samples/Creating-a-Custom-Input-Box.md

```diff
@@ -113,7 +126,10 @@ $label.Text = 'Please enter the information in the space below:'
 $form.Controls.Add($label)
 ```
 
-Add the control (in this case, a text box) that lets users provide the information you’ve described in your label text. There are many other controls you can apply besides text boxes; for more controls, see [System.Windows.Forms Namespace](https://msdn.microsoft.com/library/k50ex0x9(v=vs.110).aspx) on MSDN.
+Add the control (in this case, a text box) that lets users provide the information you’ve described
+in your label text. There are many other controls you can apply besides text boxes; for more
+controls, see [System.Windows.Forms Namespace](https://msdn.microsoft.com/library/k50ex0x9(v=vs.110).aspx)
+on MSDN.
```

Comment:
Let's fix this link (and `on MSDN` comment) while you are in this article. Update the embedded links at the bottom of the article, also.

Reply:
Fixed